### ISSUE TITLE
Above Kovan network gas limit by default

### ISSUE THREAD

**Issue by travs** [trufflesuite/truffle-contract#24]
Similar to [truffle/372](https://github.com/trufflesuite/truffle/issues/372), when using `truffle-contract` without `truffle`, my transactions get stuck in pending on the kovan chain. 

The workaround right now is to use `{gas: 4000000}` (below current Kovan gas limit) for each transaction.

It seems that the `MyContract.defaults()` returns an empty object for each of my contract abstractions, which may be part of the problem.

**Comment by tcoulter** [comment]
I'm going to close this one as Kovan upped their gas limit (as far as I know). If you run into a similar or new issue, please open a new ticket so we can address it individually. Thanks!

**Comment by stefek99** [comment]
I messing with Remix IDE and also got:

> callback contain no result Gas required exceeds limit: 3000000

Changed to `4000000` and just works.
